### DISCUSSION TITLE
Invoice template - Empty description gets filled with activity

### DISCUSSION THREAD

**Discussion by SaturnusDJ** [kimai/kimai#4060] [Questions & Answers]
Got something going on here of which I cannot find the cause. Not in the custom invoice, not in Kimai, not in the documentation.

Invoice template: One entry per time record.
Where `{{ entry.description }}` is used, when there is no description for a record, the activity is used instead. Even with `{% if entry.description is defined and entry.description != '' %}` etc. this happens. Why?

Edit: Just to be clear, in that custom invoice 'description' and 'activity' are only used once. They are not overwritten or something.

**Comment by kevinpapst** [comment]
Because that is the way how Kimai calculates these values historically:
https://github.com/kimai/kimai/issues/3967

**Reply by SaturnusDJ** [reply]
Thank you. Workaround posted in the linked issue.
